### ISSUE TITLE
Census: Text entered in other fields is not retained through out the survey

### ISSUE THREAD

**Issue by yoganandkunche** [ONSdigital/eq-survey-runner#667]
### Expected behaviour
Text entered in any of the other(Please specify) fields must be retained

### Actual behaviour
Texted entered in Other (please specify) are not retained when respondent navigates backwards

### Steps to reproduce the behaviour
Launch Census Survey
Navigate to any radio or Checkbox field with Other option
select other and enter text in the field and click save and continue
Click [Previous]

**Comment by samiwel** [comment]
Issue seems to be that other value is not persisted into the answer store.

**Comment by samiwel** [comment]
Turns out there were two issues:
1. Case sensitive match for 'other'. In the census schemas the value to be matched is 'Other'. I changed this to perform a case insensitive match when checking to see if an other value was posted.
2. When restoring the Other value an assumption was being made that the other option in the radio had the label 'Other'. This has now been changed to get the label from the schema option.
